docs/pbc/all/search/202410.0/base-shop/third-party-integrations/algolia/algolia-personalization.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Algolia Personalization
+description: Find out how you can enable Algolia personalization in your Spryker shop
+last_updated: Nov 24, 2024
+template: howto-guide-template
+---
+
+{% info_block infoBox "Info" %}
+
+Default Spryker installation supports Algolia personalization only for YVES frontend.
+If you plan to use Algolia personalization in headless frontend or mobile application follow [this guide](/docs/pbc/all/search/{{page.version}}/base-shop/third-party-integrations/algolia/algolia-personalization-headless.html). 
+
+This feature also enables other Algolia premium features:
+
+- Personalization
+- Dynamic Re-Ranking
+- Query Categorization
+- Search analytics
+- Revenue analytics
+- A/B Testing
```

Comment:
`This feature` means `Algolia Personalization` I assume `Personalization` can be removed from this list.